src/multidim/setdiff.jl

```diff
@@ -105,3 +105,25 @@ end
 #
 #     return results_list
 # end
+
+import IntervalArithmetic.hull
+doc"""
+    hull(a, b)
+
+Returns the `box hull` of the IntervalBoxes `a` and `b`, considered as
```

Comment:
Backticks (`` ` ``) are used for code, so `IntervalBox` should have backticks. "Box hull" could have normal quotes.